### DISCUSSION TITLE
What is the deal with this?

### DISCUSSION THREAD

**Discussion by babakfp** [prettier/prettier#16492] [General]
Why does code like this gets formatted like this? It's been years, and it's still like this!

![](https://github.com/user-attachments/assets/0cd7fc12-ffad-46e8-a37f-53d8d9663d73)

**Comment by lulunac27a** [comment]
There is a vertical pipe character (`|`) below the `export type FetchBaseData<TData> =` line, so that's why it's been indented by two more characters than the normal indent of four characters.

**Reply by babakfp** [reply]
Thanks for the explanation.

I expect it to be like this:

![image](https://github.com/user-attachments/assets/6b7fc6ec-1957-4937-989a-c96e163fe6e8)

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACGATwzjwDE4YwALAIQEMBnOAEXpnoB4AVNjgPjwBeADpQ8EvAB88wMZIV442bDiSz5irYw4wAro3VQ9AWwBGyzVoU72B7uhjqd2AJZQA5lcUBfb3-FJGTlArQATdnp1Xkj-MRAAGhAIDBhXaEZkUHoVCAB3AAUchEyUegAbPPpCTKSzbHowAGtKAGV6EzgAGXc4ZAAzCuY6hua2jEb3D2QYbD04JOYTVxm5hdRMZVdO2AqHbCgc1zhSwfLhkEYp8rgART0IeAGh9YArRjRW67uHp6Qzi4AR1+cAKqgwpRATAAtFA4HAwgjEiBZvRXOUpgBhCAmEz0ZBQ8rlZFXTw3ACCMFmrjMengBWUPThz3O62oMBM5QA6tRXPBGBMwHBWiU+a4AG58wgEsCMWogcXzACSUERsFaYDcqXJqtaRBuLIuGFUzC5DQwBONJ2U4r6SXczFwYPoHjxhvWE2wjoJRBIjE1rlSyON7hgXNcYRg1GQAA4AAxJbBwYGuJPO134-4vJIcMzhyPRpAAFiSBjg3HoZlO2dQ5gRiLCXXonj0LrgZBweKpUwJ9DpEBAPh8QA

`error` and `data` are indented by `4` (not `6`). The ending curly bracket of `| {` is not indented (aligns with `|`).
